3 3
48 48
Traceback (most recent call last):
  File "/home/titus.griebel/u12649/micro-sam/finetuning/specialists/training/histopathology/monuseg/evaluate_instance_segmentation_monuseg.py", line 48, in <module>
    main()
  File "/home/titus.griebel/u12649/micro-sam/finetuning/specialists/training/histopathology/monuseg/evaluate_instance_segmentation_monuseg.py", line 41, in main
    prediction_folder = run_instance_segmentation_with_decoder_inference(
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/titus.griebel/u12649/micro-sam/finetuning/specialists/training/histopathology/monuseg/evaluate_instance_segmentation_monuseg.py", line 13, in run_instance_segmentation_with_decoder_inference
    prediction_folder = run_instance_segmentation_with_decoder(
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/lustre-grete/usr/u12649/conda/envs/sam2/lib/python3.12/site-packages/micro_sam/evaluation/inference.py", line 595, in run_instance_segmentation_with_decoder
    predictor, decoder = get_predictor_and_decoder(model_type=model_type, checkpoint_path=checkpoint)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/lustre-grete/usr/u12649/conda/envs/sam2/lib/python3.12/site-packages/micro_sam/instance_segmentation.py", line 822, in get_predictor_and_decoder
    raise ValueError(f"The checkpoint at {checkpoint_path} does not contain a decoder state")
ValueError: The checkpoint at /scratch-grete/projects/nim00007/sam/models/vanilla/sam_vit_b_01ec64.pth does not contain a decoder state
